.buildkite/scripts/steps/lint.sh

```diff
@@ -11,15 +11,21 @@ node scripts/stylelint
 echo "stylelint ✅"
 
 echo '--- Lint: eslint'
+export PATH=$PATH:./node_modules/.bin
+export CI_STATS_DISABLED='true'
 # disable "Exit immediately" mode so that we can run eslint, capture it's exit code, and respond appropriately
 # after possibly commiting fixed files to the repo
 set +e;
 if is_pr && ! is_auto_commit_disabled; then
-  desc="node scripts/eslint_all_files --no-cache --fix"
-  node scripts/eslint_all_files --no-cache --fix
+#  desc="node scripts/eslint_all_files --no-cache --fix"
+#  node scripts/eslint_all_files --no-cache --fix
+  desc="moon :eslint -- --fix"
+  moon :eslint -- --fix
 else
-  desc="node scripts/eslint_all_files --no-cache"
-  node scripts/eslint_all_files --no-cache
+#  desc="node scripts/eslint_all_files --no-cache"
+#  node scripts/eslint_all_files --no-cache
+  desc="moon :eslint"
+  moon :eslint
```

Comment:
A couple other suggestions:

- @kbn/security-solution-plugin:eslint is holding one core for almost 9 minutes. 9 minutes seems a lot for a single package linting. They must have some complicated non performant rule going on
- lets try n2-standard-32 so we have more cores available and can run more projects in parallel